mta/tests/migration/controls/test_business_services.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from pytest_check import check
+
+from swagger_client.models.api_business_service import ApiBusinessService
+from swagger_client.models.api_stakeholder import ApiStakeholder
+
+
+@pytest.mark.business_service
+def test_create_business_service( create_api, get_api, delete_api, update_api ):
+#   Create a Stakeholder with email and name
+        api_stakeholder = ApiStakeholder( email="xyz@gmail.com", name="Api stakeholder")
+        new_stakeholder = create_api.stakeholders_post(api_stakeholder.to_dict())
+
+#   Create a BS with name, description and Owner
+        api_business_service = ApiBusinessService( name="Api Business", description="Api Testing", owner=new_stakeholder)
+        new_business = create_api.businessservices_post(api_business_service.to_dict())
```

Comment:
Hi @kpunwatk,
1. Please create fixtures for new stakeholder and BS
2. Use random name and email. please see https://fauxfactory.readthedocs.io/en/latest/index.html